internal/discovery/namespace_filter_test.go

```diff
@@ -53,7 +53,7 @@ func TestNamespaceFilterer_IsAllowed(t *testing.T) {
 		"match_labels_excluded_namespaces_not_allowed": {
 			namespaceLabels: labels.Set{"newrelic.com/scrape": "false"},
 			namespaceSelector: config.NamespaceSelector{
-				MatchLabels: map[string]string{
+				MatchLabels: map[string]interface{}{
```

Comment:
could we add a new test that would not pass with the old implementation? to show which use-case we are trying to cover

Reply:
Do you mean `true` as string? I added the tests in the config tests here https://github.com/newrelic/nri-kubernetes/pull/495/files#diff-78b78dc33eda1ed8d509bfa5a00f66709af57bd30d971cbaec454571a91c3e62R47